amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine
+
+RUN apk update && \
+  apk add --no-cache linux-headers build-base libxml2-dev libxslt-dev nodejs
+
+RUN pip install cwlref-runner cwltest
```

Comment:
This Dockerfile is for the image used to run the tests. see run-tests.sh below, and the README change. The reason this is being done in a Docker container is that `cwltest` has a side effect of creating a bunch of output files and folders. Performing the tests in a container gets around that problem, while also ensuring the necessary dependencies are present.